crates/matrix-sdk/src/widget/filter.rs

```diff
@@ -18,7 +18,7 @@ use ruma::events::{MessageLikeEventType, StateEventType, TimelineEventType};
 use serde::Deserialize;
 
 /// Different kinds of filters for timeline events.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, PartialEq)]
 pub enum EventFilter {
```

Comment:
This is a public type that I don't think should be comparable outside this crate (otherwise it's semver-breaking to remove this impl again that really doesn't need to exist outside of tests). Can you change to

```suggestion
#[derive(Clone, Debug)]
#[cfg_attr(test, derive(PartialEq))]
pub enum EventFilter {
```

Reply:
Same for the other two changes in this module.

Reply:
Indeed 👍 Updated.